src/components/DonationForm/DonationForm.jsx

```diff
@@ -120,35 +278,103 @@ function DonationForm() {
               <Input {...register('email2')} />
               <FormErrorMessage>{errors.email2 && errors.email2.message}</FormErrorMessage>
             </FormControl>
-          </div>
-        </div>
+          </Box>
+        </Box>
 
-        <div className={styles['field-section']}>
+        {/* <Box className={styles['field-section']}>
           <h1 className={styles.title}>Furniture Submissions</h1>
-          {furnitureFields.map((furniture, index) => (
-            // eslint-disable-next-line react/jsx-key
-            <FurnitureField index={index} register={register} removeFurniture={removeFurniture} />
+          <FormControl isInvalid={errors && errors.Item}>
+            {itemsList.map((furniture, index) => {
+              if (index === 0) {
+                // eslint-disable-next-line react/jsx-key
+                return <FurnitureField index={index} register={register} />;
+              }
+              return (
+                // eslint-disable-next-line react/jsx-key
+                <FurnitureField index={index} register={register} removeFurniture={removeItem} />
+              );
+            })}
+          </FormControl>
+        </Box>
+
+        <Box className={styles['field-section']}>
+          <Button onClick={() => appendItem({})}>Add new furniture field</Button>
+        </Box> */}
+        <Select
+          placeholder="Select Furniture"
+          value={selectedFurnitureValue}
+          onChange={ev => onSelectFurniture(ev)}
+        >
+          {furnitureOptions.map((furnitureItem, i) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <option key={i}>{furnitureItem}</option>
           ))}
-        </div>
-
-        <div className={styles['field-section']}>
-          <Button
-            onClick={() =>
-              appendFurniture({ itemName: 'Dressers', imageLink: '', description: '' })
-            }
-          >
-            Add new furniture field
-          </Button>
-        </div>
-
-        <div className={styles['field-section']}>
+        </Select>
+
+        {DonatedFurnitureList.map((donatedFurniture, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <Tag key={i}>
```

Comment:
[PENDING DESIGN DECISION] **Don't make this change quite yet**
Since you guys merged in dev already and are using `@chakra-ui/react` ver 2.4.0, you can use `Card` components instead of `Tag`.